trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/find-reports-by-year-helper.ts

```diff
@@ -0,0 +1,21 @@
+import { format, parseISO } from 'date-fns';
+import { getFormattedReportPeriodWithLongMonth } from '@ukef/dtfs2-common';
+import { UtilisationReportReconciliationSummaryItem } from '../../../types/utilisation-reports';
+import { reconciliationStatusCodeToDisplayStatus } from './reconciliation-summary-helper';
+
+type ReportViewModel = UtilisationReportReconciliationSummaryItem & {
+  formattedReportPeriod: string;
+  displayStatus: string;
+  formattedDateUploaded?: string;
+};
+
+export const getReportViewModel = (apiItem: UtilisationReportReconciliationSummaryItem): ReportViewModel => {
```

Comment:
I feel like the name of this method and the type it returns is a little vague, we display reports in different ways in different granularities so if we could make this more specific that would be better.

I'm also not very clear on how this is lining up correctly with the 'UtilisationReportsByBankAndYearViewModel' type?

Reply:
Oh yes good point, I've updated them